sessions/R-Stan-and-statistical-concepts.qmd

```diff
@@ -99,7 +100,7 @@ mod$print(line_numbers = TRUE)
 ### Take 5 minutes
 Familarise yourself with the model above.
 Do you understand all the lines?
-Which line(s) define the parameter prior distribution(s), which one(s) the likelihood, and which one(s) the data that has to be supplied to the model?
```

Comment:
Personally I prefer the original version to make it clear that they might be on separate lines.

Reply:
I've reverted this in cbacf6863a26846197b3a47e49036cd999a0edd8.